.github/workflows/mock-data.yml

```diff
@@ -0,0 +1,14 @@
+name: mock-data
+on:
+  push:
+    branches: [master, test-me-*]
+  pull_request:
+
+jobs:
+  mock-data:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - run: docker compose up --wait
```

Comment:
this is pretty slow -- it can be improved if we publish the images to github packages -- will try and do that as a followup